packages/0x.js/CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ Edit the package's CHANGELOG.json file only.
 
 CHANGELOG
 
+## v0.36.0 - TBD
+
+    * Add `zeroEx.exchange.getOrderStateAsync` to allow obtaining current OrderState for a signedOrder
+
```

Comment:
@avonian we actually don't manually update `CHANGELOG.md` (see comment at top of file). This is auto-generated from `CHANGELOG.json`. Please add the entry there!